### ISSUE TITLE
"Unexpected internal error" caused by mapping over CuArray

### ISSUE THREAD

**Issue by analogouscircuit** [JuliaGPU/GPUCompiler.jl#170]
I started experimenting with CUDA.jl yesterday and managed to produce an error with a simple example.  The error message requested that I file an issue here.  My apologies if this is caused by my doing something naive -- I'm just following orders!  A minimal example that produces the error is this:

```
module MinimalFailure

using CUDA

function badfunc!(i::Int, results::CuArray{Float32,1})
    results[i] = 0.0f0
    return
end

n=1024
data = CuArray{Float32,1}(undef,n)
idcs = CuArray(collect(1:n))
map(i -> badfunc!(i,data), idcs)

end
```

The resulting error message, system info and backtrace are below:

```
WARNING: replacing module MinimalFailure.
ERROR: LoadError: GPUCompiler.jl encountered an unexpected internal error.
Please file an issue attaching the following information, including the backtrace,
as well as a reproducible example (if possible).

InternalCompilerError: length(frames) == 1, at /home/dahlbom/.julia/packages/GPUCompiler/uTpNx/src/validation.jl:246

Compiler invocation: PTX CompilerJob of kernel broadcast_kernel(CUDA.CuKernelContext, CuDeviceArray{Nothing,1,1}, Base.Broadcast.Broadcasted{Nothing,Tuple{Base.OneTo{Int64}},Main.MinimalFailure.var"#1#2",Tuple{Base.Broadcast.Extruded{CuDeviceArray{Int64,1,1},Tuple{Bool},Tuple{Int64}}}}, Int64) for sm_61

Additional information:
 - frames = svec(svec(:jl_symbol_name_, Symbol("/buildworker/worker/package_linux64/build/src/julia.h"), 970, nothing, true, true), svec(:jl_symbol_name, Symbol("/buildworker/worker/package_linux64/build/src/rtutils.c"), 333, nothing, true, false))

Installed packages:
 - CommonSolve = v"0.2.0"
 - DelNetExperiment = v"0.1.0"
 - Requires = v"1.1.3"
 - Gtk = v"1.1.7"
 - GPUArrays = v"6.2.1"
 - ArrayLayouts = v"0.3.8"
 - CommonSubexpressions = v"0.3.0"
 - IteratorInterfaceExtensions = v"1.0.0"
 - Xorg_libXtst_jll = v"1.2.3+4"
 - Graphics = v"1.1.0"
 - GeometryBasics = v"0.2.15"
 - CodeTracking = v"1.0.5"
 - Makie = v"0.11.0"
 - Libiconv_jll = v"1.16.0+7"
 - JuliaVariables = v"0.2.4"
 - NLSolversBase = v"7.5.0"
 - FileIO = v"1.6.5"
 - Base64 = nothing
 - GLM = v"1.4.1"
 - Wayland_protocols_jll = v"1.18.0+4"
 - Colors = v"0.12.7"
 - IndirectArrays = v"0.5.1"
 - PyPlot = v"2.9.0"
 - Profile = nothing
 - PDMats = v"0.10.1"
 - WAV = v"1.1.0"
 - LibGit2 = nothing
 - PyCall = v"1.92.2"
 - VectorizationBase = v"0.16.2"
 - Graphene_jll = v"1.10.0+2"
 - IfElse = v"0.1.0"
 - OrderedCollections = v"1.4.0"
 - TimerOutputs = v"0.5.8"
 - LibVPX_jll = v"1.9.0+1"
 - Tables = v"1.4.1"
 - StatsMakie = v"0.2.3"
 - OpenSSL_jll = v"1.1.1+6"
 - ImageCore = v"0.8.21"
 - VersionParsing = v"1.2.0"
 - Ogg_jll = v"1.3.4+2"
 - CairoMakie = v"0.3.4"
 - Juno = v"0.8.4"
 - ColorVectorSpace = v"0.8.7"
 - LeftChildRightSiblingTrees = v"0.1.2"
 - CodeTools = v"0.7.1"
 - MakieLayout = v"0.9.10"
 - LNR = v"0.2.1"
 - Libepoxy_jll = v"1.5.4+1"
 - libvorbis_jll = v"1.3.6+6"
 - RecursiveFactorization = v"0.1.11"
 - MosaicViews = v"0.2.4"
 - GPUCompiler = v"0.8.3"
 - VertexSafeGraphs = v"0.1.2"
 - Xorg_xkbcomp_jll = v"1.4.2+4"
 - Combinatorics = v"1.0.2"
 - Zlib_jll = v"1.2.11+18"
 - Markdown = nothing
 - Xorg_libXcomposite_jll = v"0.4.5+4"
 - NNlib = v"0.7.11"
 - Formatting = v"0.4.2"
 - GTK3_jll = v"3.24.11+5"
 - Adapt = v"3.2.0"
 - Opus_jll = v"1.3.1+3"
 - Plots = v"1.11.2"
 - SharedArrays = nothing
 - Dbus_jll = v"1.12.16+3"
 - ProgressMeter = v"1.5.0"
 - Pidfile = v"1.2.0"
 - Missings = v"0.4.5"
 - Xorg_libXext_jll = v"1.3.4+4"
 - AssetRegistry = v"0.1.0"
 - StructArrays = v"0.4.2"
 - SpecialFunctions = v"0.10.3"
 - Observables = v"0.3.3"
 - Libtiff_jll = v"4.1.0+2"
 - RandomExtensions = v"0.4.3"
 - FuzzyCompletions = v"0.4.1"
 - DocSeeker = v"0.4.3"
 - Blosc_jll = v"1.14.3+1"
 - IOCapture = v"0.1.1"
 - ModelingToolkit = v"3.20.1"
 - GeneralizedGenerated = v"0.2.8"
 - FunctionWrappers = v"1.1.2"
 - Xorg_libXinerama_jll = v"1.1.4+4"
 - FunctionalCollections = v"0.5.0"
 - MacroTools = v"0.5.6"
 - EzXML = v"1.1.0"
 - ImageIO = v"0.2.0"
 - DiffEqCallbacks = v"2.13.5"
 - WoodburyMatrices = v"0.5.3"
 - WebIO = v"0.8.15"
 - NameResolution = v"0.1.5"
 - Test = nothing
 - PkgTemplates = v"0.7.16"
 - hicolor_icon_theme_jll = v"0.17.0+3"
 - Hexagons = v"0.2.0"
 - DataFrames = v"0.21.8"
 - BFloat16s = v"0.1.0"
 - JuliaInterpreter = v"0.8.11"
 - ThreadingUtilities = v"0.2.3"
 - FixedPointNumbers = v"0.8.4"
 - OrdinaryDiffEq = v"5.39.1"
 - LZO_jll = v"2.10.0+3"
 - FFMPEG_jll = v"4.3.1+4"
 - SHA = nothing
 - ExprTools = v"0.1.3"
 - TableTraitsUtils = v"1.0.1"
 - at_spi2_atk_jll = v"2.34.1+4"
 - xkbcommon_jll = v"0.9.1+5"
 - Hwloc = v"1.3.0"
 - RecursiveArrayTools = v"2.11.1"
 - LoopVectorization = v"0.10.0"
 - Xorg_libXfixes_jll = v"5.0.3+4"
 - Xorg_libXcursor_jll = v"1.2.0+4"
 - Parsers = v"1.1.0"
 - FreeTypeAbstraction = v"0.8.4"
 - ZygoteRules = v"0.2.1"
 - FlameGraphs = v"0.2.5"
 - Serialization = nothing
 - ForwardDiff = v"0.10.17"
 - MLStyle = v"0.4.9"
 - Xorg_libXdmcp_jll = v"1.1.3+4"
 - Grisu = v"1.0.0"
 - Fontconfig_jll = v"2.13.1+14"
 - libpng_jll = v"1.6.37+6"
 - Qt5Base_jll = v"5.15.2+0"
 - PositiveFactorizations = v"0.2.4"
 - Hiccup = v"0.2.2"
 - Xorg_xtrans_jll = v"1.4.0+3"
 - FillArrays = v"0.8.14"
 - QuadGK = v"2.4.1"
 - Zstd_jll = v"1.4.8+0"
 - MbedTLS = v"1.0.3"
 - AbstractFFTs = v"1.0.1"
 - JLLWrappers = v"1.2.0"
 - adwaita_icon_theme_jll = v"3.33.92+5"
 - ExponentialUtilities = v"1.8.1"
 - Future = nothing
 - Artifacts = v"1.3.0"
 - Xorg_xcb_util_jll = v"0.4.0+1"
 - PaddedViews = v"0.5.8"
 - Mmap = nothing
 - ParameterizedFunctions = v"5.6.0"
 - SignedDistanceFields = v"0.4.0"
 - FFMPEG = v"0.3.0"
 - Random123 = v"1.3.1"
 - Sockets = nothing
 - Xorg_libXrandr_jll = v"1.5.2+4"
 - SparseDiffTools = v"1.13.0"
 - Lazy = v"0.15.1"
 - ShiftedArrays = v"1.0.0"
 - LazyArtifacts = v"1.3.0"
 - CEnum = v"0.4.1"
 - DataAPI = v"1.6.0"
 - Statistics = nothing
 - Cairo_jll = v"1.16.0+6"
 - DiffRules = v"1.0.2"
 - CompilerSupportLibraries_jll = v"0.3.4+0"
 - METIS_jll = v"5.1.0+5"
 - DataValues = v"0.4.13"
 - NLsolve = v"4.5.1"
 - AxisAlgorithms = v"1.0.0"
 - Expat_jll = v"2.2.7+6"
 - DiffEqFinancial = v"2.4.0"
 - Sundials_jll = v"5.2.0+1"
 - CategoricalArrays = v"0.8.3"
 - AbstractTrees = v"0.3.4"
 - GenericSVD = v"0.3.0"
 - Contour = v"0.5.7"
 - Logging = nothing
 - AuditoryFilters = v"0.1.0"
 - PrettyPrint = v"0.2.0"
 - LoggingExtras = v"0.4.6"
 - ArnoldiMethod = v"0.1.0"
 - MultiScaleArrays = v"1.8.1"
 - GR_jll = v"0.57.1+0"
 - SteadyStateDiffEq = v"1.6.2"
 - CanonicalTraits = v"0.2.4"
 - libfdk_aac_jll = v"0.1.6+4"
 - Dates = nothing
 - ChainRulesCore = v"0.9.37"
 - LAME_jll = v"3.100.0+3"
 - Showoff = v"0.3.2"
 - gdk_pixbuf_jll = v"2.38.2+9"
 - Blosc = v"0.7.0"
 - FFTW_jll = v"3.3.9+7"
 - Optim = v"0.20.1"
 - HTTP = v"0.8.19"
 - Pkg = nothing
 - Xorg_libxkbfile_jll = v"1.1.0+4"
 - Interpolations = v"0.12.10"
 - FoldingTrees = v"1.0.1"
 - GLFW_jll = v"3.3.3+0"
 - Match = v"1.1.0"
 - SLEEFPirates = v"0.6.8"
 - Lz4_jll = v"1.9.2+2"
 - LightGraphs = v"1.3.5"
 - HDF5_jll = v"1.12.0+1"
 - Xorg_libX11_jll = v"1.6.9+4"
 - DataStructures = v"0.17.20"
 - Rmath_jll = v"0.2.2+2"
 - StatsFuns = v"0.9.7"
 - DataFramesMeta = v"0.6.0"
 - FiniteDiff = v"2.8.0"
 - Libffi_jll = v"3.2.1+4"
 - Compat = v"2.2.1"
 - Documenter = v"0.26.3"
 - Conda = v"1.5.1"
 - REPL = nothing
 - ATK_jll = v"2.34.1+5"
 - Xorg_libXdamage_jll = v"1.1.5+4"
 - LoweredCodeUtils = v"1.2.9"
 - AbstractPlotting = v"0.11.2"
 - FileWatching = nothing
 - MKL_jll = v"2021.1.1+1"
 - DocStringExtensions = v"0.8.4"
 - Animations = v"0.4.1"
 - Random = nothing
 - DimensionalPlotRecipes = v"1.2.0"
 - OpenSpecFun_jll = v"0.5.3+4"
 - Libglvnd_jll = v"1.3.0+3"
 - Distributions = v"0.23.8"
 - Libdl = nothing
 - BandedMatrices = v"0.15.15"
 - Xorg_xcb_util_keysyms_jll = v"0.4.0+1"
 - OpenBLAS_jll = v"0.3.9+5"
 - PooledArrays = v"0.5.3"
 - WebSockets = v"1.5.9"
 - MFCC = v"0.3.1"
 - DiffEqPhysics = v"3.9.0"
 - Xorg_xcb_util_renderutil_jll = v"0.3.9+1"
 - Xorg_libXrender_jll = v"0.9.10+4"
 - StimAnaGen = v"0.1.0"
 - DelimitedFiles = nothing
 - ResettableStacks = v"1.1.0"
 - SparseArrays = nothing
 - Ratios = v"0.4.0"
 - InvertedIndices = v"1.0.0"
 - Graphite2_jll = v"1.3.13+4"
 - PCRE_jll = v"8.42.0+4"
 - Roots = v"1.0.8"
 - UnPack = v"1.0.2"
 - Intervals = v"1.5.0"
 - SimpleTraits = v"0.9.3"
 - LaTeXStrings = v"1.2.1"
 - PyPeriModels = v"0.1.0"
 - Measures = v"0.3.1"
 - Glib_jll = v"2.59.0+4"
 - GLMakie = v"0.1.5"
 - Xorg_libXau_jll = v"1.0.9+4"
 - GridLayoutBase = v"0.3.7"
 - x264_jll = v"2020.7.14+2"
 - DifferentialEquations = v"6.16.0"
 - PlotUtils = v"1.0.10"
 - Libuuid_jll = v"2.34.0+7"
 - Atom = v"0.12.30"
 - StringDistances = v"0.10.0"
 - RecipesBase = v"1.1.1"
 - GLFW = v"3.4.1"
 - DelayDiffEq = v"5.24.1"
 - XSLT_jll = v"1.1.33+4"
 - LabelledArrays = v"1.6.0"
 - Mustache = v"1.0.10"
 - Cthulhu = v"1.6.1"
 - Pandas = v"1.4.0"
 - JpegTurbo_jll = v"2.0.1+3"
 - ImageMagick_jll = v"6.9.10-12+3"
 - ConsoleProgressMonitor = v"0.1.2"
 - Revise = v"3.1.14"
 - SafeTestsets = v"0.0.1"
 - XML2_jll = v"2.9.10+3"
 - Libgcrypt_jll = v"1.8.5+4"
 - DSP = v"0.6.10"
 - DataValueInterfaces = v"1.0.0"
 - Crayons = v"4.0.4"
 - AbstractAlgebra = v"0.11.2"
 - ColorSchemes = v"3.11.0"
 - SymbolicUtils = v"0.5.2"
 - SciMLBase = v"1.10.1"
 - Libgpg_error_jll = v"1.36.0+3"
 - DiffResults = v"1.0.3"
 - JuliaFormatter = v"0.12.3"
 - Unitful = v"1.7.0"
 - FreeType = v"3.0.1"
 - IniFile = v"0.5.0"
 - x265_jll = v"3.0.0+3"
 - TableTraits = v"1.0.1"
 - LinearAlgebra = nothing
 - MeshIO = v"0.4.0"
 - Pixman_jll = v"0.40.0+0"
 - libass_jll = v"0.14.0+4"
 - FFTW = v"1.3.2"
 - IterativeSolvers = v"0.8.5"
 - GR = v"0.57.1"
 - Xorg_xkeyboard_config_jll = v"2.27.0+4"
 - UnicodeFun = v"0.4.1"
 - Packing = v"0.4.1"
 - PNGFiles = v"0.2.1"
 - Sundials = v"4.2.6"
 - TimeZones = v"1.5.3"
 - Xorg_xcb_util_image_jll = v"0.4.0+1"
 - CommonMark = v"0.6.4"
 - InteractiveUtils = nothing
 - StatsModels = v"0.6.21"
 - Polynomials = v"1.2.1"
 - Distributed = nothing
 - RecipesPipeline = v"0.3.2"
 - SuiteSparse = nothing
 - SortingAlgorithms = v"0.3.1"
 - CSTParser = v"2.5.0"
 - nghttp2_jll = v"1.40.0+2"
 - JLD = v"0.10.0"
 - KernelDensity = v"0.5.1"
 - DiffEqNoiseProcess = v"5.5.2"
 - Wayland_jll = v"1.17.0+4"
 - TerminalLoggers = v"0.1.3"
 - OffsetArrays = v"1.6.2"
 - Reexport = v"0.2.0"
 - JSON = v"0.21.1"
 - ColorTypes = v"0.10.12"
 - Inflate = v"0.1.2"
 - Tokenize = v"0.5.15"
 - Xorg_libXi_jll = v"1.7.10+4"
 - StaticArrays = v"0.12.5"
 - Media = v"0.5.0"
 - EllipsisNotation = v"1.0.0"
 - StatsBase = v"0.32.2"
 - URIParser = v"0.4.1"
 - iso_codes_jll = v"4.3.0+4"
 - DiffEqDiffTools = v"1.7.0"
 - ArrayInterface = v"2.14.17"
 - Xorg_libpthread_stubs_jll = v"0.1.0+3"
 - Unicode = nothing
 - LineSearches = v"7.1.1"
 - ConstructionBase = v"1.1.0"
 - Pango_jll = v"1.42.4+10"
 - CoupledFields = v"0.2.0"
 - MuladdMacro = v"0.2.2"
 - BoundaryValueDiffEq = v"2.7.1"
 - BenchmarkTools = v"0.7.0"
 - Wavelets = v"0.9.2"
 - at_spi2_core_jll = v"2.34.0+4"
 - StructTypes = v"1.5.2"
 - Compose = v"0.9.2"
 - Libmount_jll = v"2.34.0+3"
 - ColorBrewer = v"0.4.0"
 - Rmath = v"0.6.1"
 - Loess = v"0.5.3"
 - CUDA = v"2.4.1"
 - ICU_jll = v"67.1.0+3"
 - LibCURL_jll = v"7.70.0+2"
 - FreeType2_jll = v"2.10.1+5"
 - MappedArrays = v"0.3.0"
 - StochasticDiffEq = v"6.25.0"
 - Xorg_libxcb_jll = v"1.13.0+3"
 - ModernGL = v"1.1.2"
 - Gettext_jll = v"0.20.1+7"
 - ShaderAbstractions = v"0.2.5"
 - Distances = v"0.10.2"
 - SuiteSparse_jll = v"5.4.0+9"
 - FriBidi_jll = v"1.0.5+6"
 - Cairo = v"1.0.5"
 - MbedTLS_jll = v"2.16.8+1"
 - Scratch = v"1.0.3"
 - LLVM = v"3.6.0"
 - Mocking = v"0.7.1"
 - IntervalSets = v"0.5.3"
 - LibSSH2_jll = v"1.9.0+3"
 - Widgets = v"0.6.2"
 - DiffEqBase = v"6.44.3"
 - TreeViews = v"0.3.0"
 - Hwloc_jll = v"2.4.1+0"
 - UUIDs = nothing
 - PoissonRandom = v"0.4.0"
 - IntelOpenMP_jll = v"2018.0.3+2"
 - NaNMath = v"0.3.5"
 - ImageMagick = v"1.2.0"
 - Latexify = v"0.14.12"
 - Calculus = v"0.5.1"
 - Xorg_xcb_util_wm_jll = v"0.4.1+1"
 - IterTools = v"1.3.0"
 - Parameters = v"0.12.2"
 - Bzip2_jll = v"1.0.6+5"
 - HDF5 = v"0.13.6"
 - HarfBuzz_jll = v"2.6.1+10"
 - Gadfly = v"1.3.2"
 - PlotThemes = v"2.0.1"
 - Printf = nothing
 - DiffEqJump = v"6.10.0"
 - ProgressLogging = v"0.1.4"
 - RandomNumbers = v"1.4.0"

Julia Version 1.5.3
Commit 788b2c77c1 (2020-11-09 13:37 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-9.0.1 (ORCJIT, broadwell)

Stacktrace:
 [1] check_ir!(::GPUCompiler.CompilerJob{GPUCompiler.PTXCompilerTarget,CUDA.CUDACompilerParams}, ::Array{Tuple{String,Array{Base.StackTraces.StackFrame,1},Any},1}, ::LLVM.CallInst) at /home/dahlbom/.julia/packages/GPUCompiler/uTpNx/src/validation.jl:246
 [2] check_ir!(::GPUCompiler.CompilerJob{GPUCompiler.PTXCompilerTarget,CUDA.CUDACompilerParams}, ::Array{Tuple{String,Array{Base.StackTraces.StackFrame,1},Any},1}, ::LLVM.Function) at /home/dahlbom/.julia/packages/GPUCompiler/uTpNx/src/validation.jl:140
 [3] check_ir!(::GPUCompiler.CompilerJob{GPUCompiler.PTXCompilerTarget,CUDA.CUDACompilerParams}, ::Array{Tuple{String,Array{Base.StackTraces.StackFrame,1},Any},1}, ::LLVM.Module) at /home/dahlbom/.julia/packages/GPUCompiler/uTpNx/src/validation.jl:131
 [4] check_ir(::GPUCompiler.CompilerJob{GPUCompiler.PTXCompilerTarget,CUDA.CUDACompilerParams}, ::LLVM.Module) at /home/dahlbom/.julia/packages/GPUCompiler/uTpNx/src/validation.jl:120
 [5] macro expansion at /home/dahlbom/.julia/packages/GPUCompiler/uTpNx/src/driver.jl:239 [inlined]
 [6] macro expansion at /home/dahlbom/.julia/packages/TimerOutputs/4QAIk/src/TimerOutput.jl:206 [inlined]
 [7] codegen(::Symbol, ::GPUCompiler.CompilerJob; libraries::Bool, deferred_codegen::Bool, optimize::Bool, strip::Bool, validate::Bool, only_entry::Bool) at /home/dahlbom/.julia/packages/GPUCompiler/uTpNx/src/driver.jl:237
 [8] compile(::Symbol, ::GPUCompiler.CompilerJob; libraries::Bool, deferred_codegen::Bool, optimize::Bool, strip::Bool, validate::Bool, only_entry::Bool) at /home/dahlbom/.julia/packages/GPUCompiler/uTpNx/src/driver.jl:39
 [9] compile at /home/dahlbom/.julia/packages/GPUCompiler/uTpNx/src/driver.jl:35 [inlined]
 [10] cufunction_compile(::GPUCompiler.FunctionSpec; kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/dahlbom/.julia/packages/CUDA/wTQsK/src/compiler/execution.jl:302
 [11] cufunction_compile(::GPUCompiler.FunctionSpec) at /home/dahlbom/.julia/packages/CUDA/wTQsK/src/compiler/execution.jl:297
 [12] check_cache(::Dict{UInt64,Any}, ::Any, ::Any, ::GPUCompiler.FunctionSpec{GPUArrays.var"#broadcast_kernel#14",Tuple{CUDA.CuKernelContext,CuDeviceArray{Nothing,1,1},Base.Broadcast.Broadcasted{Nothing,Tuple{Base.OneTo{Int64}},Main.MinimalFailure.var"#1#2",Tuple{Base.Broadcast.Extruded{CuDeviceArray{Int64,1,1},Tuple{Bool},Tuple{Int64}}}},Int64}}, ::UInt64; kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/dahlbom/.julia/packages/GPUCompiler/uTpNx/src/cache.jl:40
 [13] broadcast_kernel at /home/dahlbom/.julia/packages/GPUArrays/bjw3g/src/host/broadcast.jl:57 [inlined]
 [14] cached_compilation at /home/dahlbom/.julia/packages/GPUCompiler/uTpNx/src/cache.jl:65 [inlined]
 [15] cufunction(::GPUArrays.var"#broadcast_kernel#14", ::Type{Tuple{CUDA.CuKernelContext,CuDeviceArray{Nothing,1,1},Base.Broadcast.Broadcasted{Nothing,Tuple{Base.OneTo{Int64}},Main.MinimalFailure.var"#1#2",Tuple{Base.Broadcast.Extruded{CuDeviceArray{Int64,1,1},Tuple{Bool},Tuple{Int64}}}},Int64}}; name::Nothing, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/dahlbom/.julia/packages/CUDA/wTQsK/src/compiler/execution.jl:289
 [16] cufunction at /home/dahlbom/.julia/packages/CUDA/wTQsK/src/compiler/execution.jl:286 [inlined]
 [17] macro expansion at /home/dahlbom/.julia/packages/CUDA/wTQsK/src/compiler/execution.jl:100 [inlined]
 [18] #launch_heuristic#857 at /home/dahlbom/.julia/packages/CUDA/wTQsK/src/gpuarrays.jl:17 [inlined]
 [19] launch_heuristic at /home/dahlbom/.julia/packages/CUDA/wTQsK/src/gpuarrays.jl:17 [inlined]
 [20] copyto! at /home/dahlbom/.julia/packages/GPUArrays/bjw3g/src/host/broadcast.jl:63 [inlined]
 [21] copyto! at ./broadcast.jl:886 [inlined]
 [22] copy at /home/dahlbom/.julia/packages/GPUArrays/bjw3g/src/host/broadcast.jl:47 [inlined]
 [23] materialize(::Base.Broadcast.Broadcasted{CUDA.CuArrayStyle{1},Nothing,Main.MinimalFailure.var"#1#2",Tuple{CuArray{Int64,1}}}) at ./broadcast.jl:837
 [24] map(::Function, ::CuArray{Int64,1}) at /home/dahlbom/.julia/packages/GPUArrays/bjw3g/src/host/broadcast.jl:86
 [25] top-level scope at /home/dahlbom/research/jcuda/MinimalFailure.jl:13
 [26] include(::String) at ./client.jl:457
 [27] top-level scope at REPL[4]:1
in expression starting at /home/dahlbom/research/jcuda/MinimalFailure.jl:13

```

**Comment by analogouscircuit** [comment]
(To be clear: I see that what this does is stupid, but I'm reporting because doing it caused trouble that GPUCompiler asked to have reported.)
